modules/oe_theme_content_call_proposals/oe_theme_content_call_proposals.module

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * @file
+ * Module file is used for theming the Call for proposals content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\oe_content_call_proposals\CallForProposalsNodeWrapper;
+
+/**
+ * Implements hook_theme().
+ */
+function oe_theme_content_call_proposals_theme() {
+  return [
+    'oe_theme_content_call_proposals_publication_details' => [
+      'variables' => [
+        'date' => '',
+        'link' => [],
+      ],
+    ],
+  ];
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function oe_theme_content_call_proposals_node_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  // Run only for "Call for tenders" content type.
+  if ($entity->bundle() !== 'oe_call_proposals') {
+    return;
+  }
+
+  // Field labels are different in frontend and backend.
+  // Here we override them, in full and teaser view mode.
+  $label_overrides = [];
+  if (in_array($build['#view_mode'], ['full', 'teaser'])
+    && ($entity->get('oe_call_proposals_deadline')->count() > 1)) {
+    $label_overrides['oe_call_proposals_deadline'] = t('Deadline dates');
+  }
+
+  if ($build['#view_mode'] === 'full') {
+    $label_overrides += [
+      'oe_departments' => t('Departments'),
+      'oe_call_proposals_grants_link' => t('Results'),
+    ];
+
+    if ($entity->get('oe_departments')->count() === 1) {
+      $label_overrides['oe_departments'] = t('Department');
+    }
+  }
+
+  foreach ($label_overrides as $name => $label_override) {
+    if (!$entity->get($name)->isEmpty() && isset($build[$name]['#title'])) {
+      $build[$name]['#title'] = $label_override;
+    }
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_content_call_proposals_preprocess_field__node__oe_call_proposals_deadline__oe_call_proposals__full(&$variables, $hook) {
```

Comment:
I see this also in ternders naming.. since the field machine name already contains the content type prefix, I don't understand why we need `oe_call_proposals` at the end

```suggestion
function oe_theme_content_call_proposals_preprocess_field__node__oe_call_proposals_deadline__full(&$variables, $hook) {
```

I know its not wrong, I just don't see the point of it.

Reply:
It was implemented so for consistency. `field-node--[field_name]--[bundle]--[view_mode]` is used in another CTs